### ISSUE TITLE
No Json when there is only 1 client

### ISSUE THREAD

**Issue by edoardesd** [krylovsk/mqtt-benchmark#7]
When `--clients 1` is activated no json is displayed

**Comment by krylovsk** [comment]
Thanks, the issue is in the lack of `NaN` support in JSON (https://tools.ietf.org/html/rfc4627#section-2.4)

Will fix after #6 is merged.

**Comment by krylovsk** [comment]
(closing as duplicate of #3)
